interface/fax/fax_dispatch.php

```diff
@@ -810,7 +809,7 @@ function allCheckboxes(issel) {
 <input type='button' value='<?php echo xla('Clear All'); ?>' onclick='allCheckboxes(false)' />
 </p>
 
-<p><br /><b><?php echo xlt('Please select the desired pages to copy or forward:'); ?></b></p>
+<p><br /><strong><?php echo xlt('Please select the desired pages to copy or forward:'); ?></strong></p>
```

Comment:
The `<p>` element should have the `font-weight-bold` class and the `<strong>` tag can then be removed